quarkus/container/Dockerfile

```diff
@@ -17,7 +17,12 @@ RUN mv /tmp/keycloak/keycloak-* /opt/keycloak && mkdir -p /opt/keycloak/data
 RUN rm /opt/keycloak/lib/lib/main/com.h2database.h2*.jar  \
     /opt/keycloak/lib/lib/deployment/io.quarkus.quarkus-devservices-h2*.jar  \
     /opt/keycloak/lib/lib/main/io.quarkus.quarkus-jdbc-h2*.jar  \
-    /opt/keycloak/lib/lib/deployment/io.quarkus.quarkus-jdbc-h2-deployment*.jar
+    /opt/keycloak/lib/lib/deployment/io.quarkus.quarkus-jdbc-h2-deployment*.jar  \
```

Comment:
I don't quite understand the last (https://app.snyk.io/vuln/snyk:lic:maven:mysql:mysql-connector-java:GPL-2.0) vulnerability :-(